Added a table describing KEM background image sizes

Wriar opened the pull request:
Added a table on ``image-list-xml.html`` to describe the required resolution of separate KEM module background images.

gareth-palmer commented:
The background image sizes are already described in the first table, those are the BKEM and CP-8000-Audio, CP-8000-Video entries. That is why it says "the table above".

Perhaps it would be better to have a separate table if that wasn't obvious.

The Model column in the new table doesn't contain the actual model (eg: CP-8000-Audio) and I don't think it is necessary to describe the LCD layout or to separate the CP-8000-Audio/Video entries as the screen sizes are the same.

Wriar commented:
Fixlist:

1. KEM module image sizing was removed from the first table and moved to the second table. 
2. Additional column for 'type' was added to the second table for consistency to the first table. 
3. Removed LCD layout text in favor of consistency with the model-naming in the original table.
4. Added preview width and height in second table.

gareth-palmer commented:
The removal of the Preview Width and Height from the initial version was a good idea and should be kept. Those columns  don't make sense as my understanding is that the phone doesn't attempt to preview the expansion module background image.

A few minor changes:

1. Remove the commas that were added to the expansion module paragraph.
2. Note should be using `<b>` not `<strong>`.
3. Remove the newline between the Note and the preceding paragraph.
4. Change "Firmware 12.7(1)" to "Firmware version 12.7(1)".
5. Model is BEKEM not BEKEM (8000).
6. Each `<br>` tag should be on it's own line.

Wriar commented:
Fixlist:
1. Removed commas.
2. Added newline for ``<b>``
3. Remove newline after paragraph and before table.
4. Changed firmware text
5. Removed versioning text from BEKEM
6. Changed ``<strong`` to ``<b>``

gareth-palmer commented:
Looks good to me, thanks.
